arris_tg2492lg/connect_box.py

```diff
@@ -30,16 +31,15 @@ class ConnectBox:
     def __init__(self, websession: ClientSession, hostname: str, password: str):
         self._websession = websession
         self._hostname = hostname
-        self._password = password
+        self._password = quote(password)

```

Comment:
I would call `quote()` when creating the `arg_string` so the password stays unchanged here.